template/readme.md.js

```diff
@@ -0,0 +1,166 @@
+import { File, Text } from '@asyncapi/generator-react-sdk';
+import { getServiceClientName, getServiceClientDescription, getClientClassName, getSendOperations, getReceiveOperations, getFunctionDetails } from '../components/helpers/utils';
+
+export default function ({ asyncapi }) {
+
+  const serviceClientName = getServiceClientName(asyncapi);
+  const serviceClientDescription = getServiceClientDescription(asyncapi);
+  const clientClassName = getClientClassName(asyncapi);
+  const operations = asyncapi.operations();
+  const sendOperations = getSendOperations(operations);
+  const sendFunctions = getFunctionDetails(sendOperations);
+  const receiveOperations = getReceiveOperations(operations);
+  const receiveFunctions = getFunctionDetails(receiveOperations);
+
+  return (
+    <File name="README.md">
+      <Text>
+        ## {serviceClientName}
+      </Text>
+
+      <Text newLines={2}>
+        {serviceClientDescription}
+      </Text>
+
+      <Text newLines={2}>
+        ## Requirements
+      </Text>
+
+      <Text>
+        - Python 3.x
+      </Text>
+
+      <Text>
+        - paho-mqtt 1.6.1
+      </Text>
+
+      <Text newLines={2}>
+        ## Install Dependencies
+      </Text>
+
+      <Text>
+        We have a `requirements.txt` file under the template directory containing the `paho-mqtt` package.
+        To install the package, simply use the following:
+      </Text>
+
+      <Text>
+        ```python
+      </Text>
+
+      <Text>
+        pip install -r requirements.txt
+      </Text>
+
+      <Text>
+      ```
+      </Text>
+
+      <Text newLines={2}>
+        ## Usage of {serviceClientName}
+      </Text>
+
+      <Text newLines={2}>
+        - Import the {serviceClientName}:
```

Comment:
why you decided to change to `-` and bullet points instead of `1.` everywhere with numbered list?

Reply:
look at answer number 3 -> https://stackoverflow.com/questions/12145232/create-an-automatically-numbered-list-in-markdown-format

it happens when there are code blocks and also empty spaces between numbers - you just have to make sure there are no empty lines between points + make sure code blocks have proper indent

Reply:
Im not sure what exactly I'm doing wrong but it's not rendering, and I double-checked the indentation